### ISSUE TITLE
'front_controller_active' not updated with occ maintenance:update:htaccess

### ISSUE THREAD

**Issue by FGIKCM** [nextcloud/server#12144]
On https://github.com/nextcloud/server/blob/master/lib/private/Setup.php, when generating the **.htaccess**, the `front_controller_active` variable is always set to `true`, and not read from **config.php**...

So instead of (line 524):
```php
$content .= "\n    SetEnv front_controller_active true";
```

We should have something like:
```php
$ignoreFrontController = $config->getValue('htaccess.IgnoreFrontController', '');
...
$content .= "\n    SetEnv front_controller_active ".($ignoreFrontController ? 'false' : 'true');
```

**Comment by danielkesselberg** [comment]
Could you add some more details about your use case?  You want pretty urls but keep index.php?

**Comment by FGIKCM** [comment]
In fact I found this when trying to make **user_saml** app work (it did with NC 13.0.4, but started to fail with 13.0.5 and 14.x I think).
On my server, I configured SSO logins by X509 SSL client certificate. (I set an example Apache config in an issue on their GitHub : https://github.com/nextcloud/user_saml/issues/219)

With `front_controller_active true`, the SSO did not work, and I got an error message saying that my account is not populated, or something like this.

After digging on my Apache config, I finished by studying the .htaccess and comment each instruction one by one. And comment `front_controller_active` did the trick. So I digged anymore to se how to disable the instruction in a proper way, instead of hijacking the .htaccess :-)
